strava_to_fittrackee/s2f.py

```diff
@@ -439,6 +439,7 @@ def create_activity_from_strava(self, activity: dict, get_streams: bool = True):
                 f"Strava activity {activity_id} had no GPS data; cannot download GPX!"
             )
             get_streams = False
+            distance = 0
```

Comment:
Good catch on this one. It looks like I forgot to set `distance` in the case `get_streams` is false. Rather than defining `distance = 0` up here, can you put `distance = [0]` in the `else` block at line 478? So

```python
        else:
            latlng = [(None, None)]
            time_list = [0]
            altitude = [None]
            velocity = [None]
``` 

becomes:

```python
        else:
            latlng = [(None, None)]
            distance = [0]
            time_list = [0]
            altitude = [None]
            velocity = [None]
```